content/references/http-websocket-apis/public-api-methods/ledger-methods/ledger_entry.md

```diff
@@ -557,10 +560,172 @@ rippled json ledger_entry '{ "ticket": { "owner": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYf
 
 <!-- MULTICODE_BLOCK_END -->
 
+
 <!-- TODO: enable if/when Tickets are available on Mainnet
 [Try it! >](websocket-api-tool.html#ledger_entry-ticket)
 -->
 
+### Get Bridge Object
+
+Retrieve a [Bridge object](bridge.html) which connects a locking chain and an issuing chain. Can be provided as string (object ID of the Bridge) or as an object. _(Added by the [Sidechains amendment][])_
+
+| `Field`                 | Type                       | Description           |
+|:------------------------|:---------------------------|:----------------------|
+| `bridge`                | Object or String           | The [[Bridge object](bridge.html) to retrieve. |
+| ``          | String - [Address][]       | _(Required if `ticket` is specified as an object)_ The owner of the Ticket object. |
```

Comment:
Incomplete? The field name is missing and the description refers to tickets. You probably want four lines:

- `bridge.issuing_chain_door`
- `issuing_chain_issue`
- `locking_chain_door`
- `bridge.locking_chain_issue`